code-challenges.test.js

```diff
@@ -16,35 +16,140 @@
+       expect(endTally(votes2)).toEqual(-31)
+    })
+  })
+    // Good Error - Jest failed
+    // ReferenceError: endTally is not defined
+    // est Suites: 1 failed, 1 total
+    // Tests:       1 failed, 1 total
+
+    // 
+   
+  
+//       ReferenceError: endTally is not defined
+  
+//         61 | describe("endTally", () => {
+//         62 |     it("creates an end talley of two numbers from an object", () => {
+//       > 63 |       expect(endTally(votes1)).toEqual(11)
+//            |       ^
+//         64 |       expect(endTally(votes2)).toEqual(-31)
+//         65 |    })
+//         66 |  })
+  
+//         at Object.expect (code-challenges.test.js:63:7)
+  
+//   Test Suites: 1 failed, 1 total
+//   Tests:       1 failed, 1 total
 // b) Create the function that makes the test pass.
 
 
+const votes1 = {upVotes: 13, downVotes: 2,
+    // Expected output: 11
+    endTally: function() {
+        return this.upVotes - this.downVotes
+    // create a function within const votes using .this and the values of upVotes and downVotes
+    // return subtraction operator .upvotes minus .downvotes to receive the tally
+    }}
+const votes2 = {upVotes: 2, downVotes: 33,
+        // Expected output: -31
+
+    endTally: function() {
+        return this.upVotes - this.downVotes
+    // create a function within const votes using .this and the values of upVotes and downVotes
+    // return subtraction operator .upvotes minus .downvotes to receive the tally
+        }}
+
+    console.log(votes1.endTally())
+    console.log(votes2.endTally())
+
+// Notes: Console log returns correct ouput
+
+
+
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
+ describe("noDuplicate", () => {
+     it("returns an array with no duplicate values", () => {
+       expect(noDuplicate(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+    })
+  })
+
 const dataArray1 = ["array", "object", "number", "string", "Boolean"]
 const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
-// Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
-
+//Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+let conArray = dataArray1.concat(dataArray2);
+    console.log(conArray)
 
 // b) Create the function that makes the test pass.
+const noDuplicate = () => {
+    let conArray = dataArray1.concat(dataArray2);
+    // Merge the arrays together using .concat
+    conArray = conArray.filter((item,index) => {
+    // Use .filter to iterate through the newly merged array conArray
+    return (conArray.indexOf(item) === index) 
+    // Remove duplicates using indexOf
+})}   
+
+    
+    console.log (noDuplicate())
```

Comment:
I am seeing a lot of console logs to invoke your functions, however, this should be happening within your jest tests.